shell/app/modules/project/pages/apps/app-type-select.tsx

```diff
@@ -17,40 +17,57 @@ import './app-type-select.scss';
 import { Tooltip } from 'antd';
 import { Icon as CustomIcon } from 'common';
 import { groupBy, map } from 'lodash';
+import i18n from 'i18n';
 
 interface Img {
   src: string;
   name: string;
   value: string;
   groupIndex: number;
+  disabled: boolean;
 }
 interface IProps {
   imgOptions: Img[];
   value?: string;
   onChangeType: (value: string) => void;
 }
+
 export class AppTypeSelect extends React.PureComponent<IProps> {
   render() {
     const { imgOptions, value, onChangeType } = this.props;
     const optionGroup = groupBy(imgOptions, 'groupIndex');
+
     return (
       <div className="app-type-select">
         {map(optionGroup, (options, key) => {
           return (
             <div key={key} className="app-type-group">
-              {options.map((img) => (
-                <div
-                  key={img.name}
-                  className={classnames('img-wrapper', value === img.value && 'active')}
-                  onClick={() => onChangeType(img.value)}
-                >
-                  <img src={img.src} alt={img.name || 'image-option'} />
-                  <CustomIcon type="yuanxingxuanzhongfill" />
-                  <Tooltip title={img.name}>
-                    <div className="desc truncate">{img.name}</div>
+              {options.map((img) => {
+                const { disabled } = img;
+                const cls = classnames('img-wrapper', {
+                  active: value === img.value && !disabled,
+                  'disabled-card': disabled,
+                });
+
+                return (
+                  <Tooltip key={img.value} title={disabled ? i18n.t('dop:can-not-create-mobile-app-tip') : null}>
```

Comment:
Tip is special for mobile type app ?

Reply:
Suggest to pass a string type `disabledTip` replace disabled.

Reply:
> Tip is special for mobile type app ?

yep